src/pages/profile-page/avatar/avatar.js

```diff
@@ -82,17 +96,17 @@ const Avatar = ({ setUserImageUrl, userImageUrl, setUpload, setDeleteAvatar, t }
 
   return (
     <div className={classes.imageContainer}>
-      {userImageUrl && (
+      {userImageUrl ? (
```

Comment:
Is there any reason to change it to ternary?

Reply:
returned to previous state